koin-projects/gradle/versions-android.gradle

```diff
@@ -3,14 +3,15 @@ ext {
     android_min_version = 14
     android_target_version = 28
 
-    android_gradle_version = "3.5.3"
+    android_gradle_version = "4.2.0-alpha03"
```

Comment:
we need to check if it's not breaking anything :P

Reply:
I've never opened a PR for Koin before :sweat_smile:  
How can we validate that? I mean, what's the best way that we check and be 100% sure that everything is working fine?

Reply:
Another point is: Can this `version update` impact the version that Koin users must have in their projects?

Reply:
there is CI tooling that will run all tests. It should compile and run tests.
Also check that all samples app are running (by hand), especially the Android ones.